.circleci/config.yml

```diff
@@ -60,7 +60,8 @@ jobs:
       - run: bazel info release
 
       # Install the dependencies from NPM
-      - run: bazel run @nodejs//:yarn install
+      # TODO(gmagolan): use `bazel run :install` once bootstrap issue resolved
```

Comment:
file an issue to reference here

Reply:
which repo?

Reply:
https://github.com/bazelbuild/rules_nodejs/issues/275